app/controllers/projects/classifies_controller.rb

```diff
@@ -7,6 +7,12 @@ def show
 
     @info_request =
       @project.info_requests.where(awaiting_description: true).sample
+
+    unless @info_request
+      msg = _('There are no requests to classify right now. Great job!')
+      redirect_to @project, notice: msg
+      return
+    end
```

Comment:
Not sure I quite follow, presumably Hound is suggesting:

```suggestion
    return if @info_request
    
    msg = _('There are no requests to classify right now. Great job!')
    redirect_to @project, notice: msg
```

This seem does seem better to me.

Reply:
Ah sorry, really I should have waited for https://github.com/mysociety/alaveteli/pull/5665 before requesting review in hindsight. I've done so now and pushed.

Reply:
The problem is that we actually want to do stuff after this (i.e. set `@state_transitions`)